person.rb

```diff
@@ -20,12 +22,16 @@ def validate_name
     @name = @corrector.correct_name(name)
   end
 
+  def add_rental(book, date)
+    Rental.new(date, book, self)
+  end
+
   private
 
   def of_age?
     @age >= 18
   end
 end
 
-# st = Person.new(22,"aku")
+# st = Person.new(22,"goodMorning")
 # p st.validate_name
```

Comment:
- [ ] I would recommend removing the commented line to make the codebase cleaner.